tests/usb_phy.py

```diff
@@ -148,34 +144,10 @@ def end_test(self):
         if self._complete_fn:
             self._complete_fn(self)
 
-        # Give the DUT a reasonable time to process the packet
+        # Give the DUT a reasonable time to process the last packet
         self.wait_until(self.xsi.get_time() + self.END_OF_TEST_TIME)
 
         if self._do_timeout:
-            # Allow time for a maximum sized packet to arrive
-            timeout_time = self._clock.period_ns * 1024
-
-            if self._expect_loopback:
-                # If looping back then take into account all the data
-                total_packet_bytes = sum(
-                    [len(packet.get_bytes()) for packet in self._session.events]
-                )
-                total_data_bits = total_packet_bytes * 8
-
-                # Allow 2 cycles per bit
-                timeout_time += 2 * total_data_bits
-
-                # The clock ticks are 2ns long
-                timeout_time *= 2
-
-                # The events are copied to and from the user application
-                timeout_time *= 2
-
-            self.wait_until(self.xsi.get_time() + timeout_time)
-
-            if self._test_ctrl:
-                # Indicate to the DUT that the test has finished
-                self.xsi.drive_port_pins(self._test_ctrl, 1)
```

Comment:
Is it no longer necessary to drive the port pin?

Reply:
No, we never used this feature.